tests/performance_tests/performance_test_basic.py

```diff
@@ -30,6 +30,30 @@ class PtbTpsTestResult:
         targetTpsPerGenList: list = field(default_factory=list)
         trxGenExitCodes: list = field(default_factory=list)
 
+    @dataclass
+    class PerfTestBasicResult:
+        testStart: datetime = None
+        testEnd: datetime = None
+        testDuration: datetime = None
```

Comment:
This isn't `timedelta` type?

Reply:
Good catch. Forgot to come back and update the type in the dataclass.

Reply:
[a297634](https://github.com/AntelopeIO/leap/pull/1195/commits/a29763448abbf95fc8d541b4c5b4f5e1631eb0c2)